packages/manager/apps/web/client/app/hosting/email/detach/detach.routing.js

```diff
@@ -0,0 +1,68 @@
+import get from 'lodash/get';
+
+import component from './detach.component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.detachEmail', {
+    url: '/detachEmail',
+    component: component.name,
+    resolve: {
+      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+      pricingType: /* @ngInject */ (
+        OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS,
+      ) =>
+        OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS.PRICING_CAPACITIES.DETACH,
+      workflow: /* @ngInject */ (
+        emailOptionDetachInformation,
+        emailOptionIds,
+        serviceName,
+        OVH_MANAGER_PRODUCT_OFFERS_WORKFLOW_CONSTANTS,
+        OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS,
+      ) => ({
+        options: {
+          ...emailOptionDetachInformation[0],
+          durationToUse: `${OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS.ISO_DURATION_FORMAT.PREFIX}12${OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS.ISO_DURATION_FORMAT.MONTH}`,
```

Comment:
Can't this be handled on API side ? Where does the `12` come from ?

Reply:
Hmm API just return list of pricings, there is no information telling you what duration to use, for certain type of actions, as the duration pricings are not built correctly